ballerina/commons.bal

```diff
@@ -64,7 +64,11 @@ public isolated function getTimeInMillies(time:Civil time) returns int|Error {
     }
     time:Utc|time:Error utc = time:utcFromCivil(time);
     if utc is time:Utc {
-        return <int> decimal:round((<decimal>utc[0] + utc[1]) * 1000);
+        if utc > time:utcNow() {
+            return <int> decimal:round((<decimal>utc[0] + utc[1]) * 1000);
+        }
+        string timeInString = time.toString();
+        return error Error(string `Invalid time: ${timeInString}.`);
```

Comment:
Can't we just pass the `time` value to the string template?

Reply:
The string template expects int|float|decimal|string|boolean. But here, time is a record. So we have to convert to the string.